region/maridia/inner-green/West Sand Hall.json

```diff
@@ -843,9 +846,11 @@
       ],
       "reusableRoomwideNotable": "West Sand Hall Suitless Bootless Spring Ball",
       "note": [
-        "Perform multiple stationary lateral mid-air morphs, while avoiding the Evir projectiles, to cross the room.",
-        "It is necessary to start the jumps from the far side of the platforms in order to gain more horizontal momentum before entering the sand falls.",
-        "Falling into the sand will often result in a soft lock."
+        "The first Spring Ball jump is performed farther away from the sand tile so Samus can build up momentum and height before entering the sand fall.",
+        "The second jump must be made from the right half of the solid tile (but not the rightmost pixels.)",
+        "The thrid jump will land on the sand and must be followed by a jumnp off of the sand.",
+        "Falling into the sand will often result in a soft lock.",
+        "Only the far left section of sand can be escaped with a crouch jump."
```

Comment:
This kind of sounded like it could be confused with the left section, where the third jump takes place.
```suggestion
        "Only the far left section of sand, near the door, can be escaped with a crouch jump."
```